waiter/src/waiter/scheduler.clj

```diff
@@ -47,6 +47,11 @@
   [& args]
   `(log/log "Scheduler" :debug nil (print-str ~@args)))
 
+(defmacro logI
+  "Log InstanceTracker-specific messages."
+  [instance]
+  `(log/log "InstanceTracker" :debug nil (utils/clj->json instance)))
```

Comment:
We want to `assoc` a couple of things here:
* `leader?`
* event-type (keepalive, running, killed, healthy, etc.)

Reply:
We're not going to do the `leader?` field right now. Too much refactoring involved.